GPS next_WP
GPS cur_pos

float32 next_vel
float32 cur_vel

float32 next_heading
float32 cur_heading

int64 time
int64 gps_time

float32 battery_soc

int16 drone_priority

int64 ETA_next_WP

uint32 drone_id